cli/resolver.rs

```diff
@@ -291,6 +295,12 @@ impl NpmResolver for CliGraphResolver {
     }
   }
 
+  fn on_resolve_bare_builtin_node_module(&self, module_name: &str) {
```

Comment:
Oh, I just realized that it would have been nice to include the location where this occurred so people can easily track it down. So we would do the warning each time it occurs and for every position. I think if we update this in deno_graph then do a patch release then we won't have to release the other crates.

Reply:
PTAL https://github.com/denoland/deno_graph/pull/297